python/orca/src/bigdl/orca/automl/model/base_keras_model.py

```diff
@@ -24,17 +24,20 @@
 
 def check_tf_version():
     tf_version = tf.__version__
-    if tf_version >= "2.0":
-        raise RuntimeError(f"Currently running TensorFlow version {tf_version}. We only support"
-                           f"TensorFlow 1.x for now and has been tested on 1.15")
+    if tf_version <= "2.0":
+        raise RuntimeError(f"Currently running TensorFlow version {tf_version}. "
+                           "We only support TensorFlow 2.x for now.")
```

Comment:
I don't think we need to change this function since it is only called under `predict_with_uncertainty`, which is a function we don use.

Reply:
Ok, rollback